### PR TITLE
Updating running locally instructions

### PR DESCRIPTION
**Pull request by line47**
This is a first pass at updating the README with getting Federalist running locally. 
Addresses #1267

### CODE REVIEW

**Comment by wslack** [comment]
This seems to duplicate https://federalist-docs.18f.gov/pages/how-federalist-works/running-federalist-locally/

Should we point at one source of truth?

**Comment by line47** [comment]
I'm a big fan of the Repo README containing information about running the application locally and that as the source of truth rather than pulling that away from GitHub and having it on the docs site.

**Comment by wslack** [comment]
Right - I guess I'm asking if we should have a simultaneous PR to remove those instructions from the docs site (or just to point to the README).

File: README.md [hunk 1]
```diff
@@ -11,9 +11,81 @@ Federalist is a unified interface for publishing static government websites. It
...
+### Dependencies / Tooling
+Before you start, ensure you have the following installed:
+
+- [node](https://nodejs.org/en/download/package-manager/#osx)
+- [nvm](https://github.com/creationix/nvm#installation) or [n](https://github.com/tj/n#installation)
+- [yarn](https://yarnpkg.com/)
+- [Postgres](https://www.postgresql.org/)
+- [Docker](https://docs.docker.com/engine/installation/mac/)
+- [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
+- [AWS CLI](https://aws.amazon.com/cli/)
```
Comment (jseppi) [inline README.md]: don't really need this one actually

File: README.md [hunk 2]
```diff
@@ -11,9 +11,81 @@ Federalist is a unified interface for publishing static government websites. It
...
+
+
+### Running locally (development)
+
+1. Download or clone the `18F/federalist` repository from Github.
```
Comment (jseppi) [inline README.md]: ...from GitHub and `cd` to its directory.

File: README.md [hunk 3]
```diff
@@ -11,9 +11,81 @@ Federalist is a unified interface for publishing static government websites. It
...
+1. Run `nvm use` to ensure you are using the correct version of node
+1. Run `yarn` from the root (the directory that houses the projects files on your computer) of the repository to load modules and install Jekyll dependencies
```
Comment (jseppi) [inline README.md]: remove "from the root (the directory that houses the projects files on your computer) of the repository"

File: README.md [hunk 4]
```diff
@@ -11,9 +11,81 @@ Federalist is a unified interface for publishing static government websites. It
...
+1. Make a copy of `config/local.sample.js` and name it `local.js` and place it in the `config` folder. 
+This will be the file that holds your S3 and SQS configurations.
+1. [Register a new OAuth application on GitHub](https://github.com/settings/applications/new). You'll want to use `http://localhost:1337/auth` as the "Authorization callback url". Once you have created the application, you'll see a `Client ID` and `Client Secret`. Add these values to `config/local.js`.
+    ```
```
Comment (jseppi) [inline README.md]: in GitHub Markdown, you can "hint" the syntax of the codeblock to get specialized syntax highlighting. You do this basically by putting the file extension of what the code would be if it was in a file after the three backticks

For these samples of json, you'd do "```json"

File: README.md [hunk 5]
```diff
@@ -11,9 +11,81 @@ Federalist is a unified interface for publishing static government websites. It
...
+      }
+    }
+    ```
+1. [Register or create a new GitHub organization](https://github.com/settings/organizations). Find your organization's ID by visiting `https://api.github.com/orgs/<your-org-name>` and copying the `id` into the whitelist of `organizations` in `config/local.js`.
+    ```
+    organizations: [
+      99999999 // your org added here
+    ]
+    ```
+1. type `cf login --sso -a https://api.fr.cloud.gov -o gsa-18f-federalist -s staging` in terminal
```
Comment (jseppi) [inline README.md]: capitalize "Type"

File: README.md [hunk 6]
```diff
@@ -11,9 +11,81 @@ Federalist is a unified interface for publishing static government websites. It
...
+    organizations: [
+      99999999 // your org added here
+    ]
+    ```
+1. type `cf login --sso -a https://api.fr.cloud.gov -o gsa-18f-federalist -s staging` in terminal 
+1. Visit https://login.fr.cloud.gov/passcode to get a one time passcode
+1. Enter your passcode back into terminal
+1. Type `cf apps`
+1. Type `cf env federalist-staging` to get environment variables 
+1. Find the section in the listing of evnironment veriables that starts with `"s3": [` and look for the following values and paste those values into the S3 section in your `local.js` file.
```
Comment (jseppi) [inline README.md]: typos: `environment variables`

File: README.md (hunk 1 above)
Comment (line47) [reply README.md]: Referring to AWC CLI being removed here @jseppi ?

Comment (jseppi) [reply README.md]: yup!

File: README.md (hunk 3 above)
Comment (jseppi) [reply README.md]: (if we add the other suggestion to "`cd` to its directory")

File: README.md [hunk 7]
```diff
@@ -25,13 +25,13 @@ Before you start, ensure you have the following installed:
...
-1. Run `yarn` from the root (the directory that houses the projects files on your computer) of the repository to load modules and install Jekyll dependencies
-1. Make a copy of `config/local.sample.js` and name it `local.js` and place it in the `config` folder. 
+1. Download or clone the `18F/federalist` repository from Github and `cd` to that directory.
+1. Run `nvm use` to ensure you are using the correct version of node.
+1. Run `yarn` to load modules and install Jekyll dependencies.
+1. Make a copy of `config/local.sample.js` and name it `local.js` and place it in the `config` folder.
 This will be the file that holds your S3 and SQS configurations.
 1. [Register a new OAuth application on GitHub](https://github.com/settings/applications/new). You'll want to use `http://localhost:1337/auth` as the "Authorization callback url". Once you have created the application, you'll see a `Client ID` and `Client Secret`. Add these values to `config/local.js`.
-    ```
+    "```json"
```
Comment (jseppi) [inline README.md]: oh sorry, not with the quotation marks

**Comment by line47** [comment]
Thanks for the review @jseppi ! 👍

**Comment by line47** [comment]
@wslack I would def recommend removing the run local instructions from the doc site and just point to the README if necessary.

File: README.md [hunk 8]
```diff
@@ -11,9 +11,80 @@ Federalist is a unified interface for publishing static government websites. It
...
+
+### Running locally (development)
+
+1. Download or clone the `18F/federalist` repository from Github and `cd` to that directory.
+1. Run `nvm use` to ensure you are using the correct version of node.
+1. Run `yarn` to load modules and install Jekyll dependencies.
+1. Make a copy of `config/local.sample.js` and name it `local.js` and place it in the `config` folder.
+This will be the file that holds your S3 and SQS configurations.
+1. [Register a new OAuth application on GitHub](https://github.com/settings/applications/new). You'll want to use `http://localhost:1337/auth` as the "Authorization callback url". Once you have created the application, you'll see a `Client ID` and `Client Secret`. Add these values to `config/local.js`.
+    ```json
```
Comment (jseppi) [inline README.md]: oh shoot, sorry, I advised the wrong syntax hint, it should be `js` not `json`. apologies!

**Comment by wslack** [comment]
I opened https://github.com/18F/federalist-docs/issues/169 to track that and assigned to @line47.

**Comment by line47** [comment]
PR https://github.com/18F/federalist-docs/pull/170 address this from the docs site

**Pull request merged**